src/core/viz/expressions/sprite.js

```diff
@@ -28,7 +28,7 @@ export default class Sprite extends Base {
     constructor(url) {
         checkString('sprite', 'url', 0, url);
         super({});
-        this.type = 'sprite';
+        this.type = 'color';
```

Comment:
I think we should not change the sprite type. I would create a `COLOR_TO_SPRITE`  signature and I would add it to the `mul` expression to allow `color * sprite`, because it is actually what we are doing.

Reply:
I'll buy you that :+1:

Reply:
Changed